CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 * **[Feature]** Add authentication support for Azure Active Directory (AAD) applications. Users can now connect an [AAD tenant and AAD single tenant (AzureADMyOrg), multi-tenant (AzureADMultipleOrgs) and multi-tenant and personal Microsoft account (AzureADandPersonalMicrosoftAccount)](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps) applications.
 
+* **[Feature]** Add bring your own identity management provider support for [Auth0](https://auth0.com) and [Firebase](https://firebase.google.com/docs/auth). Developers can now connect their Auth0 and Firebase Authentication tenants and projects respectively with App Center Auth, and use the signed-in user’s identity to partition user data and send push notifications.
```

Comment:
shouldn't it be in core `App Center` directly?

Reply:
yes we should move it